package.json

```diff
@@ -137,10 +138,13 @@
     "react-tracking": "^4.2.1",
     "react-url-query": "^1.1.4",
     "sharify": "^0.1.6",
-    "styled-components": "^2.0.1",
     "superagent": "^3.6.3",
     "uuid": "^3.0.1"
   },
+  "peerDependencies": {
+    "react": "^15.5.4",
+    "styled-components": "^2.0.1"
+  },
```

Comment:
Why are these peer dependencies? I would think that they should be normal dependencies, we need the host to always include them, it’s not a suggestion.

Reply:
Ah you're right -- I've been staring at this screen for too long; updated.